modules/sdk/ant-manifest-helper/src/main/java/com/liferay/ant/manifest/helper/util/CpeUtil.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.ant.manifest.helper.util;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.ReleaseInfo;
+
+import org.apache.tools.ant.Project;
+
+/**
+ * @author Istvan Sajtos
+ * @author Drew Brokke
+ */
+public class CpeUtil {
```

Comment:
@istvansajtos I think Brian requested we use `CPEUtil` (capitalized). See https://github.com/brianchandotcom/liferay-portal/pull/159405#issuecomment-2689236611. Let's rename all `Cpe` to `CPE`.

Reply:
@drewbrokke, thanks for pointing out!